Multisig/UI/Assets/CollectiblesViewController/FlatCollectiblesListViewModel.swift

```diff
@@ -41,6 +41,7 @@ struct FlatCollectiblesListViewModel {
         guard case let .collectible(collectibleItem) = models[1], let currentPageFirstNft = collectibleItem.collectible.address else {
             return
         }
+        // drop first header if page starts with collectibles that are from the same group as last collectibles of the previous page
```

Comment:
Vitaly, come on, this just tells what the code does, but not why it is done this way.

The better comment would be:
```
// The 'header separator' items are inserted manually after downloading the collectibles. This results in duplicated headers if the same section is split between different pages of response. We have to remove such headers.
```